lib/ui/natives.dart

```diff
@@ -35,6 +35,17 @@ void _setupHooks() {
   }());
 }
 
+void saveCompilationTrace(String filePath) {
+  final result = _saveCompilationTrace();
```

Comment:
analysis is failing on this line.

Reply:
I believe it should be `final dynamic result` based on `_saveCompliationTrace`'s return type